nodejs/README.md

```diff
@@ -2,25 +2,36 @@
 
 ## Steps
 
-1. Start a locally running `mongod` version >= 4.2 running on port 27017
-2. Install dependencies with npm
+1. Clone this repository.
```

Comment:
What would you think about adding the command to clone the repository similar to: https://github.com/mongodb-university/csfle-guides/tree/master/java#steps
I am curious what others think about this since it may only cater to users that work with the command line rather than some other tool such as GitHub Desktop or manual zip archive downloads.

Reply:
I think it's fine. It's very common on other projects.

Reply:
You could also avoid the next step by mentioning something similar to the Java README referenced in the prior comment:
> Work from the repository directory for the remainder of these instructions.